src/appleseed/renderer/kernel/lighting/lighttracing/lighttracingsamplegenerator.cpp

```diff
@@ -706,6 +706,7 @@ namespace
             light_sample.m_light->sample(
                 m_shading_context,
                 light_sample.m_light_transform,
+                Vector3d(0.0),
```

Comment:
This and next change calls light's sample() function. I didn't know where to take surface point at this stage so I passed zero vector to make compiler happy.